.drone.yml

```diff
@@ -17,6 +17,26 @@ steps:
   - name: socket
     path: /var/run/docker.sock
 
+- name: docker-publish
```

Comment:
You might want to consider marking this name to reflect this is for the 'master' tag. At some point I assume we will do actual releases through here as well.

Reply:
I've renamed it to `docker-publish-master`